src/seedu/addressbook/AddressBook.java

```diff
@@ -764,6 +851,34 @@ private static void addPersonToAddressBook(String[] person) {
         ALL_PERSONS.add(person);
         savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
     }
+    
+    /**
+     * Update T2A5:
+     * Edits an existing person in the address book. Saves changes to storage file and true is returned.
+     * If specified person does not exist, false is returned.
+     *
+     * @param	person to edit
+     * @reutrn	true if specified person is found and edited, else false
+     */
+    private static boolean editPersonInAddressBook(String[] person) {
+    	final Set<String> keywords = new HashSet<>();
+    	keywords.add(person[PERSON_DATA_INDEX_NAME]);
+        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        if (personsFound.isEmpty()) {
+        	return false;
+        } else {
```

Comment:
No need for `else` as `if` has a return.  The main part of the method should be unindented if possible. Refer the section on 'guard clauses' in the handout.